src/ServiceInsight/MessageList/MessageListViewModel.cs

```diff
@@ -245,6 +251,12 @@ void TryRebindMessageList(PagedResult<StoredMessage> pagedResult)
             }
         }
 
+        void ClearResult()
+        {
+            BindResult(new PagedResult<StoredMessage>());
+            SearchBar.ClearPaging();
+        }
+
```

Comment:
Fair enough, perhaps it does indeed have some value. It may just be a matter of taste. I'm OK to have the separate method.